poolmanager/server/controlplane/plugin/noop.go

```diff
@@ -121,3 +113,13 @@ func (cp *controlPlane) RemoveRunner(lbgId string, id string) error {
 	}
 	return nil
 }
+
+func getEnv(key string) string {
+	value, ok := os.LookupEnv(key)
+	if !ok {
+		log.Panicf("Missing config key: %v", key)
+	}
+	return value
+}
+
+var ControlPlane noopControlPlane
```

Comment:
not sure about this one, it has uninitialized maps and `NewControlPlane()` returns an instance of one of these to get a handle on, perhaps this was cruft from shifting stuff around?

Reply:
probably needs to be initialized in init() which plugin.Open() would call, right?

Reply:
fixed